src/App/App.tsx

```diff
@@ -117,18 +124,41 @@ class App extends Component<Props, State> {
       comparisonCategory += '|' + cat.category
       return comparisonCategory
     }, '')
-    return list.slice(0)
+    return list
+  }
+
+  clearSelectedComparison = (): any => {
+    this.setState({ selectedComparison: {
+      id: 0,
+      category: undefined,
+      data: {
+        title: '',
+        deaths: 0,
+        image: ''
+      }
+    }})
```

Comment:
Maybe we should change category to null because it is the **intentional** absence of a value.